sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIO.java

```diff
@@ -1750,7 +1751,7 @@ public PDone apply(PCollection<TableRow> input) {
 
         // In a streaming job, or when a tablespec function is defined, we use StreamWithDeDup
```

Comment:
Updated. There's no other reference to streaming that's not related specifically to the BigQuery Streaming API